CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ option(SWIFT_INCLUDE_DOCS
     "Create targets for building docs."
     TRUE)
 
+option(SWIFT_ANALYZE_CODE_COVERAGE
```

Comment:
Huh? "`if (<variable>)`: True if the variable is defined to a value that is not a false constant".